ios/RNSScreenStack.mm

```diff
@@ -278,6 +310,11 @@ - (void)maybeAddToParentAndUpdateContainer
   }
 }
 
+- (void)addSubview:(UIView *)view
+{
+  [super addSubview:view];
+}
```

Comment:
I believe it is redundant

```suggestion
```

Reply:
Introduced in https://github.com/software-mansion/react-native-screens/pull/1802/commits/9b2e310797d78f7c83296052878f00b276a4dfcd.